app/mailers/report_mailer.rb

```diff
@@ -11,6 +11,21 @@ def deleted_user_accounts_report(email:, name:, issuers:, data:)
     mail(to: email, subject: t('report_mailer.deleted_accounts_report.subject'))
   end
 
+  def sp_issuer_user_counts_report(email:, issuer:, total:, ial1_total:, ial2_total:, name:)
+    @name = name
+    @issuer = issuer
+    @total = total
+    @ial1_total = ial1_total
+    @ial2_total = ial2_total
+    mail(to: email, subject: t('report_mailer.sp_issuer_user_counts_report.subject'))
+  end
+
+  def sp_user_counts_report(email:, data:, name:)
+    @name = name
+    attachments['sp_user_counts.csv'] = data
+    mail(to: email, subject: t('report_mailer.system_demand_report.subject'))
+  end
+
```

Comment:
this method looks unused? can we remove it?
```suggestion
```

Reply:
yeah for sure!